src/components/fields/ObjectField.js

```diff
@@ -79,8 +79,12 @@ class ObjectField extends Component {
             formContext={formContext}
           />}
         {orderedProperties.map((name, index) => {
+          const isDefault =
+            !(name in formData) ||
+            typeof Object.getOwnPropertyDescriptor(formData, name).get ===
+              "function";
```

Comment:
Here I'm trying determine whether the value in `formData` was given explicitly, or if it's just a default value from the lazily-evaluated Proxy object (this is used in the `ObjectPropertyField` component to determine whether the underlying `SchemaField` should be rendered).  It may be better to pass explicit `formData` separately from default data?